models/lung_nodule_ct_detection/configs/inference.json

```diff
@@ -13,6 +13,14 @@
     "test_datalist": "$monai.data.load_decathlon_datalist(@data_list_file_path, is_segmentation=True, data_list_key='validation', base_dir=@dataset_dir)",
     "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
     "amp": true,
+    "spatial_dims": 3,
+    "num_classes": 1,
+    "force_inferer": true,
```

Comment:
May I know why you put these 3 parameters as global vars here?

Thanks.

Reply:
Hi @Nic-Ma ,
     Because the `spatial_dims` and `num_classes` parameters are used in both `network_def` and `detector`, I think it would be better to use a parameter to control them instead of directly specifying the value in the `network_def` and `detector`.
     For the `force_inferer` (or `force_sliding_window`) parameter, it is set as a global vars so that users can easily use it in the command line, like `python -m monai.bundle run ... --force_sliding_window "True"`

Thanks,
Bin